### DISCUSSION TITLE
Make auto incement for number of rows..

### DISCUSSION THREAD

**Discussion by 22syamsudar** [rappasoft/laravel-livewire-tables#255] [Q&A]
in the datatables there is functiion call addIndexColumn/DT_RowIndex to make auto incement for number of rows...
is the any way to implement that function in this package please?

**Comment by rappasoft** [comment]
I don't understand what you mean.

**Reply by 22syamsudar** [reply]
for example...
sometimes we make like this...
@php $no=1 @endphp
@foreach($lists as $list) 
  <tr>
  <td>{{$no++}}</td>
   </tr>
@endforeach

so that, how can we implement that way in this rappasoft/laravel-livewire-tables?

**Reply by rappasoft** [reply]
Why don't you just add a column for your primary key? I don't understand why you would need that, there's no current way to do that with this package.

**Reply by 22syamsudar** [reply]
we use primary key...but sometimes we delete some records so the numbers are no longer consecutive.

**Comment by lrljoe** [comment]
This is present in v2+ use setAdditionalSelects
